CHANGES.rst

```diff
@@ -8,11 +8,26 @@ Unreleased
 
 * ``config_interpolation`` parameter for :class:`.Reddit` supporting basic and
   extended modes.
+* Methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` contains a new parameter,
+  ``fetch``, that toggles the automatic fetching of existing data from Reddit.
+  It is set to true on default.
+* Values in methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` that are left as the
+  defaults will no longer be over-written if the ``fetch`` parameter is set to
+  true, but will fill in existing values for the flair template.
+* The parameter ``text`` for methods
+  :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` is no longer required.
 
 **Removed**
 
 * Converting :class:`.APIException` to string will no longer escape unicode
   characters.
+* Methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` will no longer
+  create flairs that are using an invalid template id, but instead throw a
+  :class:`.InvalidFlairTemplateID`.
```

Comment:
Doesn't this remove the ability to create a new flair template? I may be mistaken but this seems to be the way to create them.

Reply:
No, you call the `add` method.

Reply:
Where is that method? For reference, I'm looking at https://praw.readthedocs.io/en/latest/code_overview/other/subredditflairtemplates.html but it doesn't seem to belong to that class.

Reply:
It’s a method of LinkFlairTemplates and RedditorFlairTemplates

Reply:
https://praw.readthedocs.io/en/latest/code_overview/other/subredditlinkflairtemplates.html#praw.models.reddit.subreddit.SubredditLinkFlairTemplates.add

Reply:
The superclass defines it as private method _add anyways.

Reply:
Ah, cool. Thanks!

Reply:
Maybe I should add a warning to the exception to use add when creating.

Reply:
That's a good idea.